### PR TITLE
Update FAQ and Comparison page

### PR DESCRIPTION
**Pull request by Katstodian**
Updated the LWRP FAQ (reviewed by Docs editor)
Updated Comparison Table according to new LWRP roadmap. 
Updated "Check this box" to "Tick this box".

Note for @phi-lira when merging: Changelog entry is only done in Master. I don't know how to add it for 19.1. Can you do?

### CODE REVIEW

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 1]
```diff
@@ -1,23 +1,82 @@
-# FAQ 
+# Frequently asked questions
+This section answers some frequently asked questions about the Lightweight Render Pipeline (LWRP). These questions come from the [General Graphics](https://forum.unity.com/forums/general-graphics.76/) section on our forums, from the [Unity Discord](https://discord.gg/unity) channel, and from our support teams.
 
+For information about the High Definition Render Pipeline (HDRP), please see the [HDRP documentation](https://github.com/Unity-Technologies/ScriptableRenderPipeline/wiki/High-Definition-Render-Pipeline-overview).
 
+## Can I use LWRP and HDRP at the same time?
+No. They're both built with the Scriptable Render Pipeline (SRP), but their render paths and light models are different.
+## Can I convert from one pipeline to the other?
+Yes! To do so, you'll have to re-write your Assets and redo the lighting in your game or app. You _cannot_ swap pipeline Assets from one pipeline to another at run time.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: The question should probably be. Can I convert from Built-in to LWRP. Then explain we have an upgrader that upgrades built-in shaders but custom shaders have to be upgraded manually. There's no upgrader from LWRP -> Built-in (technically it would be a downgrader :P) and also no upgrader from LWRP <-> HDRP.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 2]
```diff
@@ -1,23 +1,82 @@
...
+## How do I update the Lightweight Render Pipeline package?
+You should update via the Package Manager. In the Unity Editor, go to __Unity__ > __Window__ > __Package Manager__, and find the __Lightweight RP__ package.
 
-### Can I no longer use information from the Unity Manual?
+If you’ve added SRP code or Shader Graph manually via Github, make sure to upgrade them to the same package version as LWRP.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: via Github -> by updating manually the manifest file.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 3]
```diff
@@ -1,23 +1,82 @@
...
+## Where has Dynamic Batching gone?
 
-Because this is where you'll read specifically about LWRP. Because LWRP is a package that requires for you to download, we're keeping the docs specific to that package separate from the general user Manual. 
+The Dynamic Batching checkbox has moved from the __Player Settings__ to the [__Lightweight Render Pipeline Asset__](lwrp-asset.md).
 
-### Can I use LWRP and HDRP at the same time?
+## How do I enable Double Sided Global Illumination in the Editor?
 
-No. They're both built on the Scriptable Render Pipelines, but their render paths and light models are different. 
+In the Inspector for your Shader, find __Render Face__, and select __Both__. This means that both sides of your geometry contribute to global illumination, because LWRP doesn’t cull either side.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: In the material inspector. There's no shader inspector.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 4]
```diff
@@ -1,23 +1,82 @@
...
 
-### Can I convert from one pipeline to the other?
+Yes. The graphics quality and performance is scalable across platforms, so you can create apps for PCs and consoles as well as mobile devices.
 
-Yes! To do so, you'll have to re-write your assets and revisit your lighting strategy for your game or app.
 
+## A certain feature from the Built-in render pipeline is not supported in LWRP. Will it be?
 
+To see which features LWRP currently supports, check the [comparison table](lwrp-builtin-feature-comparison.md). 
+To see which features from the Built-in render pipeline *will* be supported in LWRP, you can check this [feature table](https://docs.google.com/spreadsheets/d/1nlS8m1OXStUK4A6D7LTOyHr6aAxIaA2r3uaNf9FZRTI/edit). LWRP will not support features marked as `Not Supported`. We intend to support features that have an open check box. Next to those, you can see information about which version of Unity we’re aiming to support them in.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: We should only reference the converted doc table here imo. The spreadsheet was just a interim thing until we didn't had the converted table.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 5]
```diff
@@ -1,23 +1,82 @@
...
-No. They're both built on the Scriptable Render Pipelines, but their render paths and light models are different. 
+In the Inspector for your Shader, find __Render Face__, and select __Both__. This means that both sides of your geometry contribute to global illumination, because LWRP doesn’t cull either side.
+## Is this render pipeline usable for desktop apps and games?
 
-### Can I convert from one pipeline to the other?
+Yes. The graphics quality and performance is scalable across platforms, so you can create apps for PCs and consoles as well as mobile devices.
 
-Yes! To do so, you'll have to re-write your assets and revisit your lighting strategy for your game or app.
 
+## A certain feature from the Built-in render pipeline is not supported in LWRP. Will it be?
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: This question is still worded weirly to me. Could we possible rephrase it to something like:
"I need a certain feature in LWRP that Built-in supports. Will LWRP support it?"

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 4 above)
Comment (phi-lira) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: "LWRP will not support features markes as `Not Supported`". Can't remember what we decided about the table. What does it show on the things we are not supporting now but will support in the future?

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 6]
```diff
@@ -1,23 +1,82 @@
...
 
+To see which features LWRP currently supports, check the [comparison table](lwrp-builtin-feature-comparison.md). 
+To see which features from the Built-in render pipeline *will* be supported in LWRP, you can check this [feature table](https://docs.google.com/spreadsheets/d/1nlS8m1OXStUK4A6D7LTOyHr6aAxIaA2r3uaNf9FZRTI/edit). LWRP will not support features marked as `Not Supported`. We intend to support features that have an open check box. Next to those, you can see information about which version of Unity we’re aiming to support them in.
+
+If a feature is marked as `In Research`, we plan to support it but don’t have a target release yet.
+
+
+## Does LWRP support a Deferred Renderer?
+Not yet. Our goal is to add support for an optimized deferred renderer in 2019.3.
+## Does LWRP have a public roadmap?
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Some questions have a space after them, some other don't. Please make it standardized.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 7]
```diff
@@ -1,23 +1,82 @@
...
+Yes. You can [check it here](https://portal.productboard.com/8ufdwj59ehtmsvxenjumxo82/tabs/3-lightweight-render-pipeline). You can add suggestions as well. To do so, you’ll have to enter your email address, but you won’t have to make an account.
+
+## Will LWRP be in LTS for 2018.4? 
+
+LWRP is still in the preview state for Long Term Service (LTS) versions of 2018 releases. We are not planning to take LWRP out of preview for 2018 LTS versions. Preview packages are not stable or ready for production, so we don’t support them in LTS versions . Use 2018 versions of LWRP at your own risk.
+## When is LWRP coming out of preview?
+LWRP will be released with 2019.1. This means we consider it stable, given the amount of features it supports, butt’s not yet feature complete. Before upgrading your project to LWRP, check our roadmap and [feature comparison table](https://docs.google.com/spreadsheets/d/1nlS8m1OXStUK4A6D7LTOyHr6aAxIaA2r3uaNf9FZRTI/edit) between LWRP and the Built-in render pipeline.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: WE should update the link here to our new table doc.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 8]
```diff
@@ -1,23 +1,82 @@
...
+## When is LWRP coming out of preview?
+LWRP will be released with 2019.1. This means we consider it stable, given the amount of features it supports, butt’s not yet feature complete. Before upgrading your project to LWRP, check our roadmap and [feature comparison table](https://docs.google.com/spreadsheets/d/1nlS8m1OXStUK4A6D7LTOyHr6aAxIaA2r3uaNf9FZRTI/edit) between LWRP and the Built-in render pipeline. 
+
+## I’ve found a bug. How do I report it?
+You can open bugs by using the [bug reporter system](https://unity3d.com/unity/qa/bug-reporting). LWRP bugs go through the same process as all other Unity bugs. You can also check the active list of bugs for LWRP in the [issue tracker](https://issuetracker.unity3d.com/product/unity/issues?utf8=%E2%9C%93&package=2&unity_version=&status=1&category=&view=hottest). 
+
+## I have an existing Project. How do I convert it from the Built-in render pipeline to LWRP?
+You can check this upgrade guide on [installing an SRP into an existing Project](installing-lwrp-into-an-existing-project.md).
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Duplicate question from the one on top "Can I convert..."

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 9]
```diff
@@ -1,23 +1,82 @@
...
+
+LWRP and the Built-in render pipeline (RP) have different quality settings. While the Built-in RP configures many settings in different places like the Quality Settings, Graphics Settings, and Player Settings, all LWRP settings are stored in the LWRP Asset. The first thing to do is to check whether your LWRP Asset settings match the settings your Built-in render pipeline Project. For example, if you disabled MSAA or HDR in your Built-in render pipeline Project, make sure they are disabled in the LWRP Asset in your LWRP Project. For advice on configuring LWRP Assets, see documentation on the [LWRP Asset](lwrp-asset.md).
+
+Also, make sure you are doing a fair comparison in terms of renderers. For this release, LWRP only supports a forward renderer, so make sure your Built-in render pipeline Project is using the forward renderer as well. 
+
+If, after comparing the settings, you still experience worse performance with LWRP, please [open a bug report](https://unity3d.com/unity/qa/bug-reporting) and attach your Project. 
+## LWRP doesn’t run on device X or platform Y. Is this expected?
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Why is it that we can say "device x or platform y" but can't say "feature x is not supported" like in a few questions above?

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 10]
```diff
@@ -1,23 +1,82 @@
...
+
+No. Please [open a bug report](https://unity3d.com/unity/qa/bug-reporting). 
+
+## My Project takes a long time to build. Is this expected?
+We are looking into how to strip Shader keywords more aggressively. You can help the Shader stripper by disabling features you don’t require for your game in the LWRP Asset. For more information on settings that affect shader variants and build time, see the [shader stripping documentation](shader-stripping.md). 
+
+## Is post-processing supported in LWRP?
+Some post-processing version 2 (PPv2) features are supported in LWRP. To see which features are supported, see [this spreadsheet](https://docs.google.com/spreadsheets/d/1nlS8m1OXStUK4A6D7LTOyHr6aAxIaA2r3uaNf9FZRTI/edit#gid=0). PPv2 doesn’t support mobile VR.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: update link to new doc table.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 11]
```diff
@@ -1,23 +1,82 @@
...
+
+We are working on many optimizations for post-processing and mobile. We’re aiming to have these in 2019.3. 
+
+## I can’t set camera clear flags with LWRP. Why?
+
+We’ve deprecated camera clear flags in LWRP. Instead, you can set the Background Type in the Camera Inspector. 
+We’ve done this because the clear flags `Depth Only` and `Don’t Care` from the Built-in render pipeline has inherent performance pitfalls. The clear flags were used for camera stacking, where one camera depends on the results of a previous camera. This is bad for performance, as it executes culling multiple times and increases bandwidth. Bandwidth cost is especially important for mobile games.
+
+For these reasons, we don’t allow camera stacking in LWRP. We are working on a solution where you can add a render pass with custom camera matrices and FOV. This way, we can provide an optimized workflow instead of creating a Camera object. We plan to expose this custom render pass in a future LWRP package.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: We should update Don't Care with whatever we decided. Also make sure we changed the name in the UI. We can push with this PR. 

This whole last paragraph is outdated now. We should say something like:

We are working on a new camera stack system for LWRP. Goal is to release it on 2019.2.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 12]
```diff
@@ -1,23 +1,82 @@
...
+
+## LWRP doesn’t render all passes in my Shader. Is this supported?
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Remove this question. We are fixing it for 2019.1

**Comment by phi-lira** [comment]
Also remove VFXManager.asset. This PR should not touch it.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 4 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: I'll inser the link to the docs page, yes.

The decision was somethign along the lines of "IF the status of a feature is left as blank in the table, it means that we intend to support it at some time."

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 5 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: I'd rather keep the current phrainsg. But can change the question to "Will LWRP support it?"

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 8 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: They're two different entry points/mindsets. So I'll keep both. :)

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 9 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Because "doesn't run on device x or platform y" is active voice, and "x is not supported" is passive.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 11 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: I won't write a goal release in the Docs. We can't - Docs are legally binding, more than the spreadsheet is.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 1 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Changed to:
## Can I convert from one pipeline to the other?

Yes! You can convert from the Built-in Unity render pipeline to LWRP. To do so, you'll have to re-write your Assets and redo the lighting in your game or app. You can use our upgrader to [upgrade Built-in Shaders to the LWRP Shaders](upgrading-your-shaders.md). For custom Shaders, you'll have to upgrade them manually. 

You _cannot_ swap pipeline Assets from one pipeline to another at run time, and there's no upgrader between LWRP and HDRP.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 6 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Does this cause any issues on your end?

**Comment by phi-lira** [comment]
You still have a VFXManager.asset being pushed to the repo.

File: com.unity.render-pipelines.lightweight/Documentation~/TableOfContents.md [hunk 13]
```diff
@@ -18,3 +18,5 @@
   * [Particles Unlit](particles-unlit-shader.md)
   * [Upgrading shaders](upgrading-your-shaders.md)
 
+* [Frequently asked questions (FAQ)](faq.md)
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/TableOfContents.md]: identing looks off here.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 14]
```diff
@@ -6,11 +6,14 @@ For information about the High Definition Render Pipeline (HDRP), please see the
 ## Can I use LWRP and HDRP at the same time?
 No. They're both built with the Scriptable Render Pipeline (SRP), but their render paths and light models are different.
 ## Can I convert from one pipeline to the other?
-Yes! To do so, you'll have to re-write your Assets and redo the lighting in your game or app. You _cannot_ swap pipeline Assets from one pipeline to another at run time.
+Yes! You can convert from the Built-in Unity render pipeline to LWRP. To do so, you'll have to re-write your Assets and redo the lighting in your game or app. You can use our upgrader to [upgrade Built-in Shaders to the LWRP Shaders](upgrading-your-shaders.md). For custom Shaders, you'll have to upgrade them manually.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Either change the question to "can I convert from built-in to LWRP" or remove Yes from the the answer.
change cannot to "should not" technically people can change from one pipeline to other at runtime.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md [hunk 15]
```diff
@@ -42,13 +43,13 @@ Yes. You can [check it here](https://portal.productboard.com/8ufdwj59ehtmsvxenju
...
+LWRP will be released with 2019.1. This means we consider it stable, given the amount of features it supports, butt’s not yet feature complete. Before upgrading your project to LWRP, check our [roadmap](https://portal.productboard.com/8ufdwj59ehtmsvxenjumxo82/tabs/3-lightweight-render-pipeline) and [feature comparison table](lwrp-builtin-feature-comparison.md) between LWRP and the Built-in render pipeline. 
 
 ## I’ve found a bug. How do I report it?
 You can open bugs by using the [bug reporter system](https://unity3d.com/unity/qa/bug-reporting). LWRP bugs go through the same process as all other Unity bugs. You can also check the active list of bugs for LWRP in the [issue tracker](https://issuetracker.unity3d.com/product/unity/issues?utf8=%E2%9C%93&package=2&unity_version=&status=1&category=&view=hottest). 
 
 ## I have an existing Project. How do I convert it from the Built-in render pipeline to LWRP?
-You can check this upgrade guide on [installing an SRP into an existing Project](installing-lwrp-into-an-existing-project.md). 
+Yes. To do so, check this upgrade guide on [installing an SRP into an existing Project](installing-lwrp-into-an-existing-project.md).
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/faq.md]: How do I convert? "Yes ..." . This doesn't make sense here.

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 11 above)
Comment (phi-lira) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Makes sense. I think it also makes sense to update the Don't Care API in this PR.

File: com.unity.render-pipelines.lightweight/Documentation~/TableOfContents.md (hunk 13 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/TableOfContents.md]: It's fine - it's because the FAQ is on the same level as other Main Headers. The indented ones you see are under SHaders in LWRP :) 

image

File: com.unity.render-pipelines.lightweight/Documentation~/faq.md (hunk 11 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/faq.md]: Martin will do it with his UI update, so it should land next week.
